backend/src/services/kanban/Jira/Jira.ts

```diff
@@ -419,18 +404,20 @@ export class Jira implements Kanban {
 
     const jiraCardHistory: JiraCardHistory = jiraCardHistoryResponse.data;
 
-    const cycleTimeInfos = Array.of<CycleTimeInfo>();
-    const originCycleTimeInfos = Array.of<CycleTimeInfo>();
-
-    Jira.getCardTimeForEachStep(jiraCardHistory, treatFlagCardAsBlock).forEach(
-      function (value, key) {
-        cycleTimeInfos.push(new CycleTimeInfo(key, value));
-      }
+    const statusChangedArray = this.putStatusChangeEventsIntoAnArray(
+      jiraCardHistory,
+      treatFlagCardAsBlock
+    );
+    const statusChangeArrayWithoutFlag = this.putStatusChangeEventsIntoAnArray(
+      jiraCardHistory,
+      false
```

Comment:
It seems like the treatFlagCardAsBlock for `originCycleTimeInfos` is the default value (`true`).
But for your refactor code, it changes to `false`.
Is this the new requirement?

Reply:
indeed, the treatFlagCardAsBlock for originCycleTimeInfos should be true. fixed that.